crates/cdk/src/nuts/nut05.rs

```diff
@@ -242,7 +242,7 @@ impl From<MeltQuoteBolt11Response> for MeltBolt11Response {
 }
 
 /// Melt Method Settings
-#[derive(Debug, Default, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq, Hash, Serialize, Deserialize)]
```

Comment:
we should add `Copy` to `MintMethodSettings` in `crates/cdk/src/nuts/nut04.rs` and then do

``` patch
-        mint_settings: self.mint_settings.clone(),
+        mint_settings: self.mint_settings,
```

everywhere

Reply:
good catch, I was thinking `MintMethodSettings` has a string so couldn't be copy but thats not the case.